add forest fires dataset

WilliamHYZhang opened the pull request:
See gist [here](https://gist.github.com/WilliamHYZhang/94255843adb87d2bc28c1baa6e0c4b7c), all tests passing.

ChanchalKumarMaji commented:
Looks good. @cyfra please add the `kokoro` tests. Thank you.

ChanchalKumarMaji commented on tensorflow_datasets/structured/forest_fires.py:
```diff
@@ -0,0 +1,119 @@
+class ForestFires(tfds.core.GeneratorBasedBuilder):
+  """Regression task aimed to predict the burned area of forest fires."""
+
+  VERSION = tfds.core.Version('2.0.0')
```
change this to `0.0.1`.

WilliamHYZhang commented:
@ChanchalKumarMaji thank you very much for your review. I added `# coding=utf-8` to the top of the files, hopefully that will fix the kokoro build error. I also changed the version to `0.0.1`. Let me know if anything else needs to be done.
